common/src/main/scala/explore/Icons.scala

```diff
@@ -286,6 +286,38 @@ object Icons {
   @JSImport("@fortawesome/pro-thin-svg-icons", "faArrowRightFromLine")
   val faArrowRightFromLine: FAIcon = js.native
 
+  @js.native
+  @JSImport("@fortawesome/pro-light-svg-icons", "faArrowsFromLine")
+  val faArrowsFromLine: FAIcon = js.native
+
+  @js.native
+  @JSImport("@fortawesome/pro-light-svg-icons", "faWrench")
+  val faWrench: FAIcon = js.native
+
+  @js.native
+  @JSImport("@fortawesome/pro-light-svg-icons", "faArrowRotateLeft")
+  val faArrowRotateLeft: FAIcon = js.native
+
+  @js.native
+  @JSImport("@fortawesome/pro-light-svg-icons", "faArrowRotateRight")
+  val faArrowRotateRight: FAIcon = js.native
+
+  @js.native
+  @JSImport("@fortawesome/pro-light-svg-icons", "faMagnifyingGlassPlus")
+  val faMagnifyingGlassPlus: FAIcon = js.native
+
+  @js.native
+  @JSImport("@fortawesome/pro-light-svg-icons", "faMagnifyingGlassMinus")
+  val faMagnifyingGlassMinus: FAIcon = js.native
+
+  @js.native
+  @JSImport("@fortawesome/pro-light-svg-icons", "faArrowsRepeat")
+  val faArrowsRetweet: FAIcon = js.native
```

Comment:
`Retweet`? Although maybe this is better since Raul has the same icon as `faArrowsRepeat` as a `pro-regular-svg-icons`.

Reply:
Ups. I was using the retweet icon but then I switched. maybe it should be fArrowsRepeatLight